plugins/modules/win_acl.ps1

```diff
@@ -122,7 +127,7 @@ ElseIf ($null -eq $inherit) {
 
 # Bug in Set-Acl, Get-Acl where -LiteralPath only works for the Registry provider if the location is in that root
 # qualifier. We also don't have a qualifier for a network path so only change if not null
-if ($null -ne $path_qualifier) {
+if (($null -ne $path_qualifier) -and ($path_qualifier -ne "Cert:")) {
```

Comment:
Is there a reason not to use the `Push-Location` semantics with the `Cert:` provider?
If so, I'd recommend explaining that in the comment above.

Reply:
The cert: provider does not use Set-Acl or Get-Acl, I will explain this in the comment.